Shreya/HTML Basics Remake/form.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+       <head>
+        <title>Form Template</title>
+       </head>
+    <body>
+        
+        <form id="form1">
+        <h1>Register</h1>
+            <p>
+                <label for="first">First Name</label>
+                <input type="text" id="first" placeholder="First Name" required>
+                <label for="last">Last Name</label>
+                <input type="text" id="last" placeholder="Last Name" required>
+            </p>
+            <p>
+                <label for="1">Gender:</label>
+              <label for="2">Male:</label>
+                <input type="radio" id="2" name="gender" required>
+                <label for="3">Female:</label>
+                <input type="radio" id="3" name="gender" required>
+                <label for="4">Others:</label>
+                <input type="radio"  id="4" name="gender" required>
+            </p>
+            <p>
+                <label for="email">Email:</label>
+                <input type="email" placeholder="Email" required>
+                <label for="password">Password:</label>
+                <input type="password" pattern=".{5,15}" title="PLease enter minimum 5 and maximum 15 characters" required>
+                <label for="date">Birthday</label>
+            <input type="date">
```

Comment:
Missing autocomplete. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)

Reply:
Still incomplete

Reply:
@aravindvnair99  you want me to put autocomplete in al the input elements?